### PR TITLE
fby35: gl: Configure GPIO interrupt function

### PR DESCRIPTION
**Pull request by BonnieLo**
Summary:
- BIC accesses the interrupt function when specific GPIOs rising/falling.

Test Plan:
Enable "IRQ_SML1_PMBUS_ALERT_N" and "FM_DBP_PRESENT_N" to check if BIC accesses interrupt function successfully.

Test Log:
    uart:~$ platform gpio list_all IRQ_SML1_PMBUS_ALERT_N
    [41 ] IRQ_SML1_PMBUS_ALERT_N             : OD  | input (O) | 0(0)
    uart:~$ platform gpio set val 41 1
    access IRQ_SML1_PMBUS_ALERT_N (Add debug message to check if BIC accesses interrupt function)
    gpio[41] --> 1 ,success!
    uart:~$ platform gpio list_all IRQ_SML1_PMBUS_ALERT_N
    [41 ] IRQ_SML1_PMBUS_ALERT_N             : OD  | input (I) | 1(1)

    uart:~$ platform gpio list_all FM_DBP_PRESENT_N
    [44 ] FM_DBP_PRESENT_N                   : OD  | input (O) | 0(0)
    uart:~$ platform gpio set val 44 1
    access FM_DBP_PRESENT_N (Add debug message to check if BIC accesses interrupt function)
    gpio[44] --> 1 ,success!
    uart:~$ platform gpio list_all FM_DBP_PRESENT_N
    [44 ] FM_DBP_PRESENT_N                   : OD  | input (I) | 1(1)

### CODE REVIEW

File: meta-facebook/yv35-gl/src/platform/plat_isr.c [hunk 1]
```diff
@@ -13,14 +13,443 @@
...
+	}
+	if (k_work_cancel_delayable(&SLP3_work) != 0) {
+		LOG_ERR("[%s] Failed to cancel delayable work", __func__);
+	}
+}
+
+K_WORK_DELAYABLE_DEFINE(set_DC_on_5s_work, set_DC_on_delayed_status);
+K_WORK_DELAYABLE_DEFINE(set_DC_off_10s_work, set_DC_off_delayed_status);
+/*
+ * TODO :
```
Comment (garnermic) [inline meta-facebook/yv35-gl/src/platform/plat_isr.c:94]: Please remove todo comments.

File: meta-facebook/yv35-gl/src/platform/plat_isr.c [hunk 2]
```diff
@@ -13,14 +13,443 @@
...
+
+		if (k_work_cancel_delayable(&set_DC_off_10s_work) != 0) {
+			LOG_ERR("Cancel set dc off delay work fail");
+		}
+		set_DC_off_delayed_status();
+	} else {
+		set_DC_on_delayed_status();
+		k_work_schedule(&set_DC_off_10s_work, K_SECONDS(DC_OFF_10_SECOND));
+
+		/*
```
Comment (garnermic) [inline meta-facebook/yv35-gl/src/platform/plat_isr.c:113]: Please remove todo comments.

**Review by garnermic — CHANGES_REQUESTED** [review]

File: meta-facebook/yv35-gl/src/platform/plat_isr.c (hunk 1 above)
Comment (JeffShih-wiwynn) [reply meta-facebook/yv35-gl/src/platform/plat_isr.c]: Todo comments are removed. Please check, thanks.

File: meta-facebook/yv35-gl/src/platform/plat_isr.c (hunk 2 above)
Comment (JeffShih-wiwynn) [reply meta-facebook/yv35-gl/src/platform/plat_isr.c]: Todo comments are removed. Please check, thanks.

**Comment by garnermic** [comment]
This PR conflicts with #880. Can you rebase the two and re-submit this PR?

**Pull request closed without merging**